watertap/examples/flowsheets/lsrro/tests/test_lsrro.py

```diff
@@ -498,10 +502,10 @@ def optimization_data(self, model):
         fs = model.fs
 
         data[fs.product.flow_mass_phase_comp[0,'Liq','H2O']]   = 0.732036
-        data[fs.product.flow_mass_phase_comp[0,'Liq','NaCl']]  = 0.445058e-3
-        data[fs.disposal.flow_mass_phase_comp[0,'Liq','H2O']]  = 0.197964
-        data[fs.disposal.flow_mass_phase_comp[0,'Liq','NaCl']] = 0.695545e-1
-        data[fs.costing.LCOW]   = 1.51258
+        data[fs.product.flow_mass_phase_comp[0,'Liq','NaCl']]  = 0.448848e-3
+        data[fs.disposal.flow_mass_phase_comp[0,'Liq','H2O']]  = 0.197967
+        data[fs.disposal.flow_mass_phase_comp[0,'Liq','NaCl']] = 0.695512e-1
+        data[fs.costing.LCOW]   = 1.46933
```

Comment:
Just want to make sure the changes in LCOW are only attributed to reduction in electricity cost (added operating cost for pressure exchanger which will be a negative value)